index.html

```diff
@@ -45,7 +45,7 @@
 
       <ul class="design-section-links">
         <li><a href="./wide-copy.html">Mobile First</a></li>
-        <li><a>Responcive Design</a></li>
+        <li><a>Responsive Design</a></li>
```

Comment:
You can also comment on individual line numbers or chunks of code for more clarity.